### DISCUSSION TITLE
DAI 2024-2025 - My Docker Compose application - Thiebaud Jonathan

### DISCUSSION THREAD

**Discussion by JONATHAN-THIEBAUD** [heig-vd-dai-course/heig-vd-dai-course#415] [Show and tell]
https://github.com/JONATHAN-THIEBAUD/java-ios-docker

![image](https://github.com/user-attachments/assets/e62ec254-48c6-4e21-a0e6-d3159386bc07)

**Comment by ludelafo** [comment]
Thank you!

You can always find the solution in the solution repository if you ever need to compare your work with the official one here: https://github.com/heig-vd-dai-course/heig-vd-dai-course-solutions.

Do not hesitate to ask questions if something is not clear, either in this discussion or in the Q&A category of the GitHub Discussions here: https://github.com/orgs/heig-vd-dai-course/discussions/categories/q-a.
